scrapper.py

```diff
@@ -25,39 +34,133 @@ class Crawler:
+        self.article_id = article_id
+        self.article = Article(article_url, article_id)
+
+    def _fill_article_with_meta_information(self, article_bs):
+        title_bs = article_bs.find('h1')
+        self.article.title = title_bs.text
+
+        self.article.author = 'NOT FOUND'
+
+        self.article.topics = 'NOT FOUND'
+
+        date_bs = article_bs.find('div', class_='news__date').text
+        months = {"января": "01", "февраля": "02", "марта": "03", "апреля": "04",
+                  "мая": "05", "июня": "06", "июля": "07", "августа": "08",
+                  "сентября": "09", "октября": "10", "ноября": "11",
+                  "декабря": "12"}
+        for month in months:
+            if month in date_bs:
+                date_bs = date_bs.replace(month, months[month])
+        self.article.date = datetime.strptime(date_bs, '%H:%M, %d %m %Y')
+
+    def _fill_article_with_text(self, article_bs):
+        texts_bs = article_bs.find('div', class_='article__body')
+        paragraphs_bs = texts_bs.find_all('p', class_=None)
+        self.article.text = ''
+        for paragraph_bs in paragraphs_bs:
+            self.article.text += paragraph_bs.text
+
+    def parse(self):
+        response = requests.get(self.article_url)
+        article_bs = BeautifulSoup(response.text, 'lxml')
+
+        self._fill_article_with_text(article_bs)
+        self._fill_article_with_meta_information(article_bs)
+        return self.article
+
 
 
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
-
+    path = pathlib.Path(base_path)
+    if path.exists():
+        shutil.rmtree(base_path)
+    path.mkdir(parents=True, exist_ok=True)
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r', encoding='utf-8') as file:
+        config = json.load(file)
+        max_articles = config["total_articles_to_find_and_parse"]
+        seed_urls = config["seed_urls"]
+
+        if not seed_urls:
+            raise IncorrectURLError
+        if not isinstance (seed_urls, list):
+            raise IncorrectURLError
+        for article_url in seed_urls:
+            correct_url = re.match(r'https://', article_url)
+            if not correct_url:
+                raise IncorrectURLError
+
+        if not isinstance (max_articles, int):
+            raise IncorrectNumberOfArticlesError
+
+        if max_articles <= 0:
+            raise IncorrectNumberOfArticlesError
+
+        if max_articles > 200:
+            raise NumberOfArticlesOutOfRangeError
 
+        return seed_urls, max_articles
 
 if __name__ == '__main__':
     # YOUR CODE HERE
-    pass
+    seed_links, maximum_articles = validate_config(CRAWLER_CONFIG_PATH)
+    prepare_environment(ASSETS_PATH)
+    crawler = Crawler(seed_urls=seed_links, max_articles=maximum_articles)
+    crawler.find_articles()
+
+    COUNTER = 1
```

Comment:
in python, we usually use capital letters to denote constants 
you are changing the value of `COUNTER ` after each iteration, so it is not exactly constant 
I would recommend to use `enumerate()` for declaring a variable that would serve as an ascending index